### ISSUE TITLE
Restarting PHP 8.0 service

### ISSUE THREAD

**Issue by trovster** [shamoon/cloudways-api-action#1]
The documentation talks about restarting `php5-fpm` service which would have an operation of `service.php5-fpm.restart`. However, I need to restart PHP 8.0.

When manually restarting a service using the Cloudways UI, the page makes a JSON request to `https://platform.cloudways.com/api/v1/service/state` with `service "php8.0-fpm"`.

I tried the operation of `service.php8.0-fpm.restart` but it gave the following error;

```
Found Cloudways API operation: 'service'
Error: Operation parameters not understood
```

My guess is the split on `.` which is causing an issue. How would I resolve this?

**Comment by shamoon** [comment]
Hmm, I can take a look at this in the next few days but the Cloudways docs don’t list that as an option for `service` https://developers.cloudways.com/docs/#!/ServiceApi%23changeServiceState

Did you experiment with any different values for this (perhaps directly against the API)?

**Comment by shamoon** [comment]
Also, did you try `php5-fpm`? If I recall correctly, you can’t run more than one PHP version on a cloudways server anyway so this just may be the alias they use for restarting all PHP services. For example, I do not think the ”5” actually refers to the PHP version.

**Comment by trovster** [comment]
I see the list on the API documentation matches the list you have for this action. I was unaware that `php5-fpm` would restart PHP FPM in general, I guess I could try that.

I haven't experimented with other values against the API directly. I only need to restart this one service on deployment and found this action which looked like it would work. I noticed the `php8.0-fpm` service name in the AJAX call when restarting the service manually using the UI.

**Comment by shamoon** [comment]
Alright, I confirmed what youre reporting, the docs are just wildly inaccurate and the correct service is in fact `php8.0-fpm` (or `php7.4-fpm`, etc). I forgot they have an API playground to easily verify this https://developers.cloudways.com/play/#!/Service/Change_Service_State 

I just released v1.0.6 which should fix this, hopefully!

Thanks for reporting
